virtme_ng/run.py

```diff
@@ -733,33 +731,31 @@ def checkout(self, args):
     def config(self, args):
         """Perform a make config operation on a kernel source directory."""
         arch = args.arch
-        cmd = "virtme-configkernel --defconfig"
+        cmd = ["virtme-configkernel", "--defconfig"]
         if args.verbose:
-            cmd += " --verbose"
+            cmd.append("--verbose")
         if not args.force and not args.kconfig:
-            cmd += " --no-update"
+            cmd.append("--no-update")
         if arch is not None:
             if arch not in ARCH_MAPPING:
                 arg_fail(f"unsupported architecture: {arch}")
             arch = ARCH_MAPPING[arch]["qemu_name"]
-            cmd += f" --arch {arch}"
+            cmd += ["--arch", arch]
         user_config = str(Path.home()) + "/.config/virtme-ng/kernel.config"
         if os.path.exists(user_config):
-            cmd += f" --custom {user_config}"
+            cmd += ["--custom", user_config]
         if args.config:
             for conf in args.config:
-                cmd += f" --custom {conf}"
+                cmd += ["--custom", conf]
         if args.configitem:
             for citem in args.configitem:
-                cmd += f" --configitem {citem}"
+                cmd += ["--configitem", citem]
         # Propagate additional Makefile variables
-        for var in args.envs:
-            cmd += f" {var} "
+        if args.envs is not None:
```

Comment:
Is None ever possible?

The original code would also fail in case of `args.envs == None`.
```
for var in None:
```
raises an exception:
`TypeError: 'NoneType' object is not iterable`

Reply:
Arf, I changed this in my last version, and forgot to commit. Indeed, the condition is useless, `args.envs` is always set, but can be empty. I will remove this line and another one.